### DISCUSSION TITLE
Documentation on processor example

### DISCUSSION THREAD

**Discussion by jnoche** [hneemann/Digital#603] [Q&A] [answered]
I will soon be teaching an undergraduate course on microprocessors, and I am planning on using the processor example in Digital.  I understand that you also have the Assembler software (ASM 3) that interfaces with it, and I have used it.  I see that ASM 3 has a short description of the instruction set.  But aside from this, is there any other documentation for the processor example?  (For example, something that explains what the signal names mean.)

**Comment by hneemann** [comment, marked as answer]
I have material for a whole lecture series, which describes this processor and also explains all the details. 
Unfortunately, it's all only available in German, as I'm not teaching any international courses at the moment. I am sorry.

**Reply by jnoche** [reply]
Okay.  Thanks, anyway.
